frontend/src/modules/products/components/EditProductForm.tsx

```diff
@@ -25,6 +27,19 @@ export const EditProductForm: React.FC<EditProductFormProps> = ({
       },
     });
   };
+  const getUser = useGlobalStore((state) => state.getUser);
+  const companyType = getUser()?.company?.type;
+
+  const isSuppManu = () => {
+    if (
+      companyType === CompanyDTOTypeEnum.Supplier ||
+      CompanyDTOTypeEnum.Manufacturer
+    ) {
+      return true;
+    } else {
+      return false;
+    }
+  };
```

Comment:
we can simplify this to just be
```suggestion
  const isSuppManu = () => {
    return companyType === CompanyDTOTypeEnum.Supplier || companyType === CompanyDTOTypeEnum.Manufacturer
  };
```

also, we can't just do:
```javascript
companyType === CompanyDTOTypeEnum.Supplier || CompanyDTOTypeEnum.Manufacturer
```
like you had it because what that condition is saying is `companyType === CompanyDTOTypeEnum.Supplier` (which correctly checks if the company type is `SUPPLIER`) OR `CompanyDTOTypeEnum.Manufacturer` (which is just checking if this value is defined). instead, we need to do like in my suggestion above:
```javascript
companyType === CompanyDTOTypeEnum.Supplier || companyType === CompanyDTOTypeEnum.Manufacturer
```
this way, we check if `companyType === CompanyDTOTypeEnum.Supplier` (which checks if company type is `SUPPLIER`) OR `companyType === CompanyDTOTypeEnum.Manufacturer` (which checks if company type is `MANUFACTURER`